src/commands/start.yml

```diff
@@ -15,10 +15,8 @@ parameters:
     description: Whether to install the `awslocal` command line interface into the build environment (requires python3 and pip3)
 
 steps:
-  - run:
-      name: Install LocalStack
-      command: |
-        test "<<parameters.install-awslocal>>" != "true" || pip3 install -q localstack awscli-local[ver1]
+  - tools:
```

Comment:
nitpick: 
I can't comment on the parameter, which is why I post this comment a few lines below.
I think - since you reverted the renaming of the parameter and mentioned that you want to do this in the next iteration - there should be a TODO indicating that this should be changed with the next major version.
And again, I think it would also be okay to just create a new major version of the orb if you prefer to rename the param with this PR.

Reply:
I'd not leave it as a TODO here for now as you've pointed out before it would be nice to have all the tool options in a single (perhaps enum) parameter. So leaving the comment there would indicate we want to keep this option the way it is just correct the naming.